README.md

```diff
@@ -6,12 +6,19 @@ A Discord bot made for ACM Cyber & Psi Beta Rho. :)
 0. Install Node and make sure corepack is enabled (`corepack enable`).
 1. Download and copy `.env.example` as `.env`
 2. Run `pnpm install` to install dependencies
-3. Either ask me for your own discord bot user OR Create your own discord bot application: https://discordjs.guide/preparations/setting-up-a-bot-application.html
-4. [Invite your discord bot to our shared testing discord server](https://discordjs.guide/preparations/adding-your-bot-to-servers.html#creating-and-using-your-invite-link). If you need admin, let me (Alec) know.
+3. Either ask me (Alec or Andrew) for your own discord bot user OR Create your own discord bot application: https://discordjs.guide/preparations/setting-up-a-bot-application.html
```

Comment:
We should probably avoid hard-coded names here - maybe like "the infrastructure lead?"